bark/world/world.hpp

```diff
@@ -190,7 +193,7 @@ class World : public commons::BaseType {
   double world_time_;
   AgentRTree rtree_agents_;
   bool remove_agents_;
-  double frac_lateral_offset_;
+  double lateral_difference_threshold_;
```

Comment:
As a side note: I think this should be a map specific feature. Why do we not persist this in the map_interface?

Reply:
Mhh, I think it can even be a behavior specfic feature since it affects when different agents consider other vehicles as merging onto their lane.